src/foam/doc/CodeTab.js

```diff
@@ -0,0 +1,12 @@
+foam.CLASS({
```

Comment:
This model isn't being used. You can remove it and any requires of it and nothing should change.